setup.py

```diff
@@ -51,7 +51,7 @@
     "pytest==5.3.2",  # MIT license
     "pytest-cov==2.8.1",  # MIT
     "pytest-watch==4.2.0",  # MIT
-    "xain-sdk==0.4.0",  # Apache License 2.0
+    "xain-sdk @ git+https://github.com/xainag/xain-sdk.git@development#egg=xain_sdk-0.3.0",  # Apache License 2.0
```

Comment:
it seems weird that xain-sdk 0.4 has been released but we create an egg with version 0.3 from the development branch. Does it matter at all?

Reply:
We have tried to this once but it seems to be that pip can't figure out if something was updated on a git repository. That means, if we change the version number, pip won't clone the the repository again. Nevertheless, I agree it seems weird. I will change it to `0.5.0`.